api/src/main/java/org/apache/iceberg/view/SQLViewRepresentation.java

```diff
@@ -18,14 +18,17 @@
  */
 package org.apache.iceberg.view;
 
+import edu.umd.cs.findbugs.annotations.Nullable;
```

Comment:
Can you remove this? We don't use nullable annotations.

Reply:
since `SQLViewRepresentation` is an Immutable, this indicates to the builder that certain fields are not required and can indeed be null when an instance is constructed

Reply:
Nullable annotations have been used in puffin and metrics, but I do think we should standardize on `javax.annotation.Nullable` as opposed to this findbugs version if we are going to be explicit about nullability.

Reply:
Makes sense. There's still a few other places in the Iceberg code base which are using the umd.cs instead of javax. It make sense to use the javax one, it looks like the umd.cs one is deprecated? I'll create a tracking issue so that we can update to use `javax` in the remaining parts of the code